src/Components/CreateImageWizardV2/CreateImageWizard.tsx

```diff
@@ -57,15 +59,27 @@ const CreateImageWizard = () => {
             id="step-image-output"
             footer={
               <CustomWizardFooter
-                disableNext={
-                  useAppSelector((state) => selectImageTypes(state)).length ===
-                  0
-                }
+                disableNext={targetEnvironments.length === 0}
               />
             }
           >
             <ImageOutputStep />
           </WizardStep>
+          <WizardStep
```

Comment:
So on the existing wizard we don't show the target environment step, not even in the sidebar, as long as they haven't selected anything that needs additional sharing details.

Don't think it's a blocker but we might ned to change that at some point.

Reply:
nvm I just saw the first isHidden :facepalm: